### PR TITLE
Update copyright to Northern.tech and current year.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: BSDmakefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (C) CFEngine AS
+#  Copyright 2017 Northern.tech AS
```
Comment (jimis) [inline BSDmakefile]: Why do you delete the old copyright? Who had copyright before 2017? Does the official company name contain a dot?

I would expect something like:

```
Copyright 2017-present Northern Tech AS
Copyright 2008-2017 CFEngine AS
```

Comment (kacf) [inline BSDmakefile]: I got the string directly from @ThomasRyd. This is the one we should use.

Comment (kacf) [inline BSDmakefile]: Oh, and retaining copyright for CFEngine is definitely wrong, the company doesn't exist anymore. All its copyrights were transferred to Northern.tech.

Comment (jimis) [inline BSDmakefile]: This is the first source code that I see that deletes copyright history and mentions copyright only for the current year like it just got created.

**Comment by kacf** [comment]
Passed [here](https://ci.cfengine.com/job/testing-enterprise-pr/343/).

**Comment by kacf** [comment]
Cherry-picked to 3.7.x, 3.10.x and 3.11.x as well.

File: Makefile.am [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (C) CFEngine AS
+#  Copyright 2017 Northern.tech AS
 #
 #  This file is part of CFEngine 3 - written and maintained by CFEngine AS.
```
Comment (olehermanse) [inline Makefile.am]: @kacf `written and maintained by CFEngine AS` ?

Comment (kacf) [inline Makefile.am]: Right, good catch. I'll fix that.

**Pull request merged**